src/ts/ConvivaAnalytics.ts

```diff
@@ -509,14 +519,19 @@ export class ConvivaAnalytics {
   };
 
   private onVideoQualityChanged = (event: VideoQualityChangedEvent) => {
-    if (!this.isSessionActive()) {
-      return;
-    }
     // We calculate the bitrate with a divisor of 1000 so the values look nicer
     // Example: 250000 / 1000 => 250 kbps (250000 / 1024 => 244kbps)
     const bitrateKbps = Math.round(event.targetQuality.bitrate / 1000);
-    console.warn('go video quality changed ', this.sessionKey, bitrateKbps);
 
+    if (!this.isSessionActive()) {
+      // Since the first videoPlaybackQualityChanged event comes before playback ever started we need to store the
+      // value and use it for tracking when initializing the session.
+      // TODO: remove this workaround when the player event order is fixed
```

Comment:
The player event order probably won't be fixed. It selects a bitrate before playback starts, and it can't do it the other way around - because with which bitrate would it start then?

Reply:
Hm but it feels weird that a `*Playback*Changed` event is fired without active `playback`.